spec/features/user_visits_homepage_spec.rb

```diff
@@ -4,7 +4,7 @@
   scenario "successfully" do
     visit root_path
 
-    expect(page).to have_css("h1", text: "Welcome to Tutorify")
+    expect(page).to have_css("h3", text: "Sign In")
```

Comment:
rather than testing for `Sign In` I think a better test will be to check the presence of "Tutorify" in the nav bar